src/RowWiseSparseAdagradFused.cc

```diff
@@ -63,45 +65,58 @@ class GenRowWiseSparseAdagradFused {
   // The hash depends on embedding dimension (block size), prefetch distance,
```

Comment:
nit: can you update the comment that the first key in code cache is pointer to avx2 mask?